component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1042,37 +1044,41 @@ protected boolean retryAuthenticationEnabled() {
     /**
      * Get the connection and proceed with SMS API's rest call.
      *
-     * @param httpConnection  the connection
-     * @param context         the authenticationContext
-     * @param headerString    the header string
-     * @param payload         the payload
-     * @param httpResponse    the http response
-     * @param encodedMobileNo the encoded mobileNo
-     * @param smsMessage      the sms message
-     * @param otpToken        the token
-     * @param httpMethod      the http method
+     * @param httpConnection        the connection
+     * @param context               the authenticationContext
+     * @param headerString          the header string
+     * @param payload               the payload
+     * @param httpResponse          the http response
+     * @param receivedMobileNumber  the encoded mobileNo
+     * @param smsMessage            the sms message
+     * @param otpToken              the token
+     * @param httpMethod            the http method
```

Comment:
Lets format this properly. Sentences need to start from a upper case letter and end with a `.`. Shall we fix it?

Reply:
addressed with [0b521b08e51e783e0b8f316a3e0135a80e20534f](https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/124/commits/0b521b08e51e783e0b8f316a3e0135a80e20534f)